netsim/tools/nuts/inventory/hosts.j2

```diff
@@ -0,0 +1,16 @@
+{% for nn,nd in nodes.items()
+     if nd.ansible_connection|default('') in ['ssh','paramiko','network_cli']
+       and ('ipv4' in nd.mgmt or 'ipv6' in nd.mgmt) %}
+{{ nn }}:
+  hostname: "{{ nd.mgmt.ipv4|default(nd.mgmt.ipv6) }}"
+  username: "{{ nd.ansible_user }}"
+  password: "{{ nd.ansible_ssh_pass }}"
```

Comment:
There's also the option to use Vagrant-generated SSH key on FRR VMs (not sure whether that's relevant). The path to the key (when it exists) is in **ansible_ssh_private_key_file** and it's relative to the current (lab) directory, so starting with `.vagrant/`